src/Dotnet.Script.DependencyModel/Environment/ScriptEnvironment.cs

```diff
@@ -55,16 +55,6 @@ private ScriptEnvironment()
 
         public bool IsNetCore => _netCoreVersion.Value != DotnetVersion.Unknown;
 
-        public void OverrideTargetFramework(string targetFramework)
```

Comment:
the override should stay if you want to have something different than `net472` (which can be the hardcoded default)
for example for OmniSharp `net472` makes sense, but you can use/host this library in a different context too, so therefore it makes sense to provide a workaround

Reply:
I've reintroduced the framework override. 
Note that the override is no longer used in the Desktop tests as it conflicts with the new compilation dependency tests there. Still able to override, it is just that we use the default for the tests now which is `net472`

Reply:
yep that's fine